### DISCUSSION TITLE
ibm-i-development-pack: Add to readme and/or documentation

### DISCUSSION THREAD

**Discussion by SJLennon** [codefori/vscode-ibmi#526] [Ideas]
I ran across this extension by accident.  Shouldn't it be mentioned in the readme.  I don't think they all get installed automatically, but I can't tell from my installation.

**Comment by worksofliam** [comment]
The development pack really is just a collection of extensions that I like to use for IBM i development. We could mention it in the docs and/or readme also.

**Comment by SJLennon** [comment]
I brought this up because I noticed someone had an issue getting snippets working because he didn't know about the RPGLE language tools (vscode-rpgle) extension.  He'd checked the documentation on snippets and it doesn't mentions the need for the extension (probably got overtaken by the splitting up of the original code).  I'll fix that section of the documentation to note that the vscode-rpgle extension is needed.  And think about where/how to mention the development pack.
